src/V3Ast.h

```diff
@@ -1632,6 +1632,9 @@ class AstNode VL_NOT_FINAL {
     inline bool isDouble() const;
     inline bool isSigned() const;
     inline bool isString() const;
+    virtual string dotColor() const { return "black";}
+    virtual string dotShape() const { return "ellipse";}
+    virtual string dotStyle() const { return "bold";}
```

Comment:
There are no overloads of this currently so let's just use the constants where these are called. If we later want to add more styling, we can do it in `AstNode::dumpTreeDot` directly, rather than adding yet more virtual functions distributing this info over a lot of lines.